.gitignore

```diff
@@ -2,4 +2,5 @@ node_modules
 saucelabs
 testingbot
 browserstack
+selenium-standalone
```

Comment:
It's the directory created by `SeleniumTunnel#download()`. I assume that's why the above directories are there.